scripts/pi-hole/js/groups-domains.js

```diff
@@ -512,7 +512,7 @@ function addDomain() {
   domains = domains.filter(function (el) {
     return el !== "";
   });
-  const domainStr = utils.escapeHtml(JSON.stringify(domains));
+  const domainStr = JSON.stringify(domains);
```

Comment:
I think this should stay escaped. You are reverting what you added with commit https://github.com/pi-hole/web/pull/2851/commits/bc1c6bc117eb88adb0c519b8ea9f24725ca755d3

Reply:
See my comment above, toasts now always escape themselves